lib/src/facts/freebsd/dmi_resolver.cc

```diff
@@ -14,7 +14,7 @@ namespace facter { namespace facts { namespace freebsd {
         result.bios_version = kenv_lookup("smbios.bios.version");
         result.bios_release_date = kenv_lookup("smbios.bios.reldate");
         result.uuid = kenv_lookup("smbios.system.uuid");
-        result.serial_number = kenv_lookup("smbios.planar.serial");
```

Comment:
No.

Well kinda?

It's the Planar serial - likely the MB serial.  Almost certainly NOT what you want.